docs/notes/bugfix-21803.md

```diff
@@ -0,0 +1,3 @@
+# Added documentation note for the rename command that it can overwrite 
+existing files
```

Comment:
@livecodesam Line 1 has probably a line break, and this will result in the release note to appear like this:

<img width="879" alt="Screen Shot 2019-06-26 at 09 41 20" src="https://user-images.githubusercontent.com/5111835/60157119-b40cd680-97f6-11e9-9918-aea82b475975.png">